sbnobj/Common/CRT/classes_def.xml

```diff
@@ -1,6 +1,7 @@
 <lcgdict>
 
-  <class name="sbn::crt::CRTHit" ClassVersion="14">
+  <class name="sbn::crt::CRTHit" ClassVersion="15">
+   <version ClassVersion="15" checksum="3264936168"/>
```

Comment:
While the pull request states that the dictionary for the association is added, this change suggests that the `sbn::crt::CRTHit` object is also changed. But this pull request is not adding any change to it, so its checksum (and version) should not change.
Please clarify.

Reply:
https://github.com/SBNSoftware/sbnobj/commit/d7c9ab34265bc902b795c951a716e346e2d78738#diff-fdbc5cb8037ead542ffc2d38998a6785bc87d5eb1eb58c8a5e28ee5a2f5ab2a5

I have added this in to the CRTHIt.hh

Reply:
Thank you.